py3status/modules/diskdata.py

```diff
@@ -79,6 +87,39 @@ class Py3status:
     }
     unit = "B/s"
 
+    class Meta:
+        deprecated = {
+            "remove": [{"param": "sector_size", "msg": "obsolete parameter"}],
+            "rename": [
+                {"param": "disk", "new": "disks", "msg": "rename parameter to `disks`"}
+            ],
+        }
+
+    def _resolve_disk_name(self, path):
```

Comment:
(Fluffy) Can you shuffle the methods so we keep `post_config_hook` (init) on top and `diskdata` (main) at bottom? Thx.

Reply:
moved private methods on bottom, `post_config_hook` is already on top of `diskdata` (or I did not understand)

Reply:
`post_config_hook`, private methods, `diskdata`, `on_click` (not used in this module)

Reply:
so different than master version? Ok for me, maybe you could document those requirements :P